conda_smithy/configure_feedstock.py

```diff
@@ -1266,6 +1265,66 @@ def _github_actions_specific_setup(
     jinja_env, forge_config, forge_dir, platform
 ):
 
+    # Handle GH-hosted and self-hosted runners runs-on config
+    # Do it before the deepcopy below so these changes can be used by the
+    # .github/worfkflows/conda-build.yml template
+    runs_on = {
+        "osx-64": {
+            "os": "macos",
+            "self_hosted_labels": ("macOS", "x64"),
+        },
+        "osx-arm64": {
+            "os": "macos",
+            "self_hosted_labels": ("macOS", "arm64"),
+        },
+        "linux-64": {
+            "os": "ubuntu",
+            "self_hosted_labels": ("linux", "x64"),
+        },
+        "linux-aarch64": {
+            "os": "ubuntu",
+            "self_hosted_labels": ("linux", "ARM64"),
+        },
+        "linux-ppc64le": {
+            "os": "ubuntu",
+            "self_hosted_labels": ("linux", ""),
```

Comment:
I don't understand the empty label

Reply:
Oops, I left that as a placeholder because I wanted to see if I could find the PPC label on the github/runner repo but turns out it's not supported, so we need to remove it.